schemas/bashly.json

```diff
@@ -334,6 +334,17 @@
           "type": "boolean",
           "default": true
         },
+        "validate": {
+          "title": "validate",
+          "description": "A validation function for the current environment variable\nhttps://bashly.dannyb.co/configuration/environment-variable/#validate",
+          "type": "string",
```

Comment:
Thank you. I applied the correction in the strings.json. As for the bashly.json, I am leaving it as is. Rationale:
1. the bashly.json is generated from the [support/schema/bashly.yml](https://github.com/DannyBen/bashly/blob/add/environment-variable-validation/support/schema/bashly.yml)
2. the new `validate` in `environment_variables` is a copy-paste from the same function in `args` and `flags`
3. I prefer these schemas to be a *guide* rather than being a complete lockdown on the exact requirements. Easier to maintain.

If you feel strongly otherwise, I might accept a separate PR with changes to bashly.yml rather than the JSON artifact.